### ISSUE TITLE
Не подхватывает цену из-под Woo Price Calculator

### ISSUE THREAD

**Issue by DaddyDorset** [artikus11/art-woo-order-one-click#8]
В проекте к товарам прикручен WPC Pro. Он аяксом меняет цену продукта в зависимости от работающей формулы. Но при нажатии на кнопку заказа в форму попадает базовая цена продукта, а не сформированная на странице продукта в результате вычислений калькулятора.

Плюс, что, впрочем, понятно, в [awooc-hidden], никак не попадают поля и значения калькулятора, привязанного к продукту. Нет их ни в заказе, ни в почте, соответственно. Я так понимаю, опции товара этим плагином могу учитываться только в случае создания вариативного товара?

**Comment by artikus11** [comment]
Да, про такой плагин в первый раз слышу и скорее всего с ним работать и не будет. Я просто не знаю как данный плагин работает
